### PR TITLE
readme enhancement for nuget package

### LINKED ISSUE #148: Add links to preview release NuGet packages
Add links to preview release (v8.0.0-preview*) NuGet packages in the README.md or on the release description page.

### PR DESCRIPTION
**Pull request by WeihanLi**
**Closes #148 **

## PR Checklist

- [x] Created a feature/dev branch in your fork (vs. submitting directly from a commit on main)
- [x] Based off latest main branch of toolkit
- [x] PR doesn't include merge commits (always rebase on top of our main, if needed)
- [ ] Tested code with current [supported SDKs](../#supported)
- [ ] New component
  - [ ] Pull Request has been submitted to the documentation repository [instructions](../blob/main/Contributing.md#docs). Link: 
  - [ ] Added description of major feature to project description for NuGet package (4000 total character limit, so don't push entire description over that)
- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Header has been added to all new source files (run _build/UpdateHeaders.bat_)
- [x] Contains **NO** breaking changes
- [ ] Every new API (including internal ones) has full XML docs
- [ ] Code follows all style conventions

## Other information

Add the NuGet package info and links, thanks for @Nirmal4G 's help

Preview: <https://github.com/WeihanLi/dotnet-CommunityTookit/blob/readme-nuget-package-enhancement/README.md#-what-does-this-repo-contain>

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,12 @@
...
-- [`CommunityToolkit.Common`](https://docs.microsoft.com/dotnet/api/?term=communitytoolkit.common): a set of helper APIs shared with other CommunityToolkit libraries.
+Package | Latest stable | Latest Preview | Description
+---------|---------------|---------------|------------
+[`CommunityToolkit.Common`](https://docs.microsoft.com/dotnet/api/?term=communitytoolkit.common) | [![CommunityToolkit.Common](https://img.shields.io/nuget/v/CommunityToolkit.Common)](https://nuget.org/packages/CommunityToolkit.Common/) | [![CommunityToolkit.Common](https://img.shields.io/nuget/vpre/CommunityToolkit.Common)](https://nuget.org/packages/CommunityToolkit.Common/absoluteLatest) | A set of helper APIs shared with other `CommunityToolkit` libraries.
```
Comment (Nirmal4G) [inline README.md:16]:
```suggestion
[`CommunityToolkit.Common`](https://docs.microsoft.com/dotnet/api/?term=communitytoolkit.common) | [![CommunityToolkit.Common](https://img.shields.io/nuget/v/CommunityToolkit.Common)](https://nuget.org/packages/CommunityToolkit.Common/) | [![CommunityToolkit.Common](https://img.shields.io/nuget/vpre/CommunityToolkit.Common)](https://nuget.org/packages/CommunityToolkit.Common/absoluteLatest) | A set of helper APIs shared with other [`CommunityToolkit`](https://docs.microsoft.com/windows/communitytoolkit) libraries.
```

**Review by Nirmal4G — CHANGES_REQUESTED** [review]

**Comment by Nirmal4G** [comment]
@Sergio0694 @michael-hawker Also, the `docs.microsoft.com` has been changed to `learn.microsoft.com`. Do we need to update it here or in a separate PR?

**Review by Nirmal4G — APPROVED** [review]
🎊

**Review by Sergio0694 — APPROVED** [review]
Thank you! 🙂

**Comment by Sergio0694** [comment]
@Nirmal4G The docs have redirects, so we can update them later on if we want. We're using docs everywhere anyway 😄

**Comment by Nirmal4G** [comment]
I was aware of that. I only discovered through this PR. Just notifying the project maintainers. That's all. 😇

**Pull request merged**